CHANGELOG.asciidoc

```diff
@@ -77,6 +77,12 @@ https://github.com/elastic/beats/compare/v6.6.0...6.x[Check the HEAD diff]
 *Journalbeat*
 
 - Add missing journalbeat non breaking fixes. {pull}9106[9106]
+- Use `log.source.address` instead of `log.source.ip` for network input sources. {pull}9487[9487]
+- Rename many `redis.log.*` fields to map to ECS. {pull}9315[9315]
+- Rename many `icinga.*` fields to map to ECS. {pull}9294[9294]
+- Rename many `postgresql.log.*` fields to map to ECS. {pull}9303[9303]
+- Rename many `kafka.log.*` fields to map to ECS. {pull}9297[9297]
```

Comment:
These don't look like from the same PR. Should these be removed?

Reply:
Good catch, yes, they should be removed. Fixing...

Reply:
Cleaned up in 525ebd4a3754631b9fa3825c1de2cb4d95ae2d6e.